doc/docker.md

```diff
@@ -0,0 +1,72 @@
+# Docker
+
+To have the smallest possible changes between a deployment in pure
+Ansible and a container based one, docker images are built from
+ansible roles.
+
+The `role2docker` script in the `bin` directory creates the docker
+image for a specific role. For example: `./bin/role2docker wazo_message_bus`.
```

Comment:
I'ld specify the base OS here, something like this. Imho relevant since you also go down to the detail of specifying the packages in the labels.

> The `role2docker` script in the `bin` directory creates the eponym docker
> image, applying the given role over Debian Buster official image. For example: `./bin/role2docker > wazo_message_bus`.

Reply:
done